README.md

```diff
@@ -1,11 +1,83 @@
 # Introduction
 
-Welcome to the public prototype repository for [Cranq](https://cranq.io)!
+Welcome to the public prototype repository for [Cranq](https://cranq.io)! This repository contains the official prototype set for CRANQ, developed & curated by us.
 
-# Documentation
+If you are looking for a general documentation of CRANQ, check out our dedicated [cranq-tutorials](https://github.com/Cranq-io/cranq-tutorials) repository!
 
-Check out our dedicated [cranq-tutorials](https://github.com/Cranq-io/cranq-tutorials) repository!
+# Structure
 
-# Contributing
+Upon the installation of the CRANQ IDE, this repository is copied to your home folder:
+```
+~/.cranq/repository
+```
+Every leaf directory in the folder structure represents a CRANQ prototype, structured by it's ***namespace*** - for example, the description of the node ```data/array/Item deleter``` can be found under ```./data/array/item deleter```.
 
-Coming soon
+> **_How do I know the namespace of a node?_**
+>
+> The fully qualified name of a CRANQ node consists of it's name, and it's namespace path. Namespaces are divided by forward slashes ```/```, with the last segment being the node name.  
+> For example:  
+> ```data/array/Item deleter```  
+> -> ```data```: root namespace  
+> --> ```array```: sub-namespace  
+> ---> ```Item deleter```: node name  
+
+
+# Public namespaces
+
+This repository contains the public namespaces. These root namespaces are reserved & maintained by the CRANQ development team.
+
+> **_Caution_**
+>
+> These are maintained by the CRANQ IDE installer, any changes to them might get ***overwritten*** on upgrade or re-install:
+> - Custom prototypes created in the reserved namespaces
+> - Changes to the public prototypes  
+> 
+> For your own work, use ***Private namespaces*** instead.
+
+
+# Private namespaces
+
+By convention, private namespaces should follow the ```#{name}``` naming convention - these will be ignored during upgrade/install.
+The installer will create a couple of these by default:
+```
+#team
+#user
+#workspace
+```
+Currently, any new node created through the CRANQ IDE will be automatically placed in the ```#workspace``` namespace. 
+
+> **_How do I set/change the namespace of my node?_**
+>
+> Just rename your prototype in the CRANQ IDE, with the namespace path you with to use:
```

Comment:
Typo: "you with to use" -> "you wish to use"

Reply:
Fixed